tests-lit/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_custom_target(tests-lit)
+
+set(CLANG_EXEC ${PATH_TO_LLVM}/bin/clang++)
+set(FILECHECK_EXEC filecheck)
```

Comment:
What is the best way to check / ensure that `filecheck` exists and throw an error otherwise?

Reply:
I think the 'right' way is to use [find_program](https://cmake.org/cmake/help/latest/command/find_program.html), but it's up to you.

Reply:
As discussed in person, we leave it with `set`.